### ISSUE TITLE
Negative time values catched by whitespace_011

### ISSUE THREAD

**Issue by erick166** [jeremiah-c-leary/vhdl-style-guide#922]
**Environment**
VHDL Style Guide (VSG) version: 3.13.1

**Describe the bug**
Negative time values like `-1 ns` are reported by this rule.

**To Reproduce**
```
p_test : process is
  variable v_timestamp : time := -1 ns;
begin

  if v_timestamp /= -1 ns then
    -- do something
  end if;

end process p_test;
```

**Expected behavior**
I would expect -1 ns to be a valid assignment, and not - 1 ns.

**Comment by jeremiah-c-leary** [comment]
Morning @erick166,

I pushed an update to this issue to the `issue-922` branch.  When you get a chance could you check it out on your end and let me know if it works on your end?

Thanks,

--Jeremy

**Comment by erick166** [comment]
Hi,

The negative value in the **if** statement is okay now, but negative values in the declaration part are still reported. E.g.

```
constant C_MIN_RANGE : integer := -2 ** 8;
variable v_timestamp : time := -1 ns;
```

**Comment by jeremiah-c-leary** [comment]
Morning @erick166,

I pushed another fix for this to the branch.  When you get a chance could you check it out and let me know how it works for you?

Thanks,

--Jeremy

**Comment by erick166** [comment]
Hi,

It works now with the new fix, thanks!
